packages/flutter/test/material/time_picker_test.dart

```diff
@@ -861,6 +861,14 @@ void _testsInput() {
     expect(find.text(helpText), findsOneWidget);
   });
 
+  testWidgets('Hour and minute label text is used - Input', (WidgetTester tester) async {
+    const String hourLabelText = 'Custom hour label';
+    const String minuteLabelText = 'Custom minute label';
+    await mediaQueryBoilerplate(tester, true, entryMode: TimePickerEntryMode.input, hourLabelText: hourLabelText, minuteLabelText: minuteLabelText );
+    expect(find.text(hourLabelText), findsOneWidget);
+    expect(find.text(minuteLabelText), findsOneWidget);
+  });
```

Comment:
I think we could make this two tests instead of one, so we may detect individual regressions.

Reply:
Done, thank you for the feedback